app/Http/Requests/Tasks/CreateRequest.php

```diff
@@ -24,10 +24,20 @@ public function authorize()
      */
     public function rules()
     {
-        return [
-            'name'        => 'required|max:60',
-            'description' => 'nullable|max:255',
-            'progress'    => 'required|numeric|max:100',
+        $rules = [
+            'name'     => 'required|max:60',
+            'progress' => 'required|numeric|max:100',
         ];
+
+        //Allow for flexibility instead of optionless hard-coded value for "description". This is
+        //achieved using environmental variable.
+        //A value of zero (0) will mean "no limit"
+
+        $char_len_task_description = intval(env('CHAR_LEN_TASK_DESCRIPTION', 255));
```

Comment:
Change this to **DESCRIPTION_CHAR_LEN**, too. And change variable name to **camelCase**.